python/tk_framework_desktopserver/certificates.py

```diff
@@ -46,12 +48,41 @@ def exists(self):
         """
         return os.path.exists(self._cert_path) and os.path.exists(self._key_path)
 
-    def remove_files(self):
+    def backup_files(self):
         """
-        Removes the files from the
+        Backups the certificate files to the backup/year-month-day-hour-minute-second folder.
         """
-        os.unlink(self._cert_path)
-        os.unlink(self._key_path)
+        parent_folder = os.path.dirname(self._cert_path)
```

Comment:
The calling method already has extensive logging around what it does. I don't believe this is needed here.